tests/observations.test.js

```diff
@@ -0,0 +1,251 @@
+import path from "path";
+import { readJsonSync } from "fs-extra";
+
+/* File To Check */
+const DATA_PATH = path.join(
+  __dirname,
+  "..",
+  "public",
+  "data",
+  "observations.json"
+);
+
+/* Read And Parse JSON File To An Object */
+const { observations } = readJsonSync(DATA_PATH);
+
+/* Global and Spillover Tests */
+// Global score and rank
+test("If Algeria have a correct SDG index score and rank", () => {
+  const getSdgIndex = (number) =>
+    observations.find((obs) => obs.id === `${number}-TOT`);
+
+  expect(getSdgIndex(66)).toMatchObject({
+    s: 70.86,
+    r: 66,
+  });
```

Comment:
Better to test with the actual dataId of Algeria. (Otherwise, we are only testing that dataId 66 has rank 66, but we don't know if it's Algeria or not)

```suggestion
  const algeria = regions.find(region => region.slug === "algeria")

  const getSdgIndex = (number) =>
    observations.find((obs) => obs.id === `${number}-TOT`);

  expect(getSdgIndex(algeria.dataId)).toMatchObject({
    s: 70.86,
    r: 66,
  });
```